### PR TITLE
merge-window-anchor Created WindowAggregate by merging anchor aggregate and window

### PR DESCRIPTION
**Pull request by vigbk**

### CODE REVIEW

File: blurr/core/window_data_group.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    def __init__(self, fully_qualified_name: str,
+                 schema_loader: SchemaLoader) -> None:
+        super().__init__(fully_qualified_name, schema_loader)
+        self.window_value = self._spec[self.ATTRIBUTE_WINDOW_VALUE]
+        self.window_type = self._spec[self.ATTRIBUTE_WINDOW_TYPE]
+        self.source = self.schema_loader.get_schema_object(
+            self._spec[self.ATTRIBUTE_SOURCE])
+
+
+class _Window:
```
Comment (indradeep) [inline blurr/core/window_data_group.py]: Why is this needed as a separate object?  Why can this not follow the same pattern as we have in session aggregate?

Comment (prashastk) [reply blurr/core/window_data_group.py]: Explained offline to Tito. He is convinced that its needed. Will rename to WindowSource.
